### DISCUSSION TITLE
Password is never strong enough

### DISCUSSION THREAD

**Discussion by IzMichael** [supabase/supabase#2620] [Questions] [answered]
Hi! I'm trying to create my first database, but whatever password I use, it says it's not strong enough. I've tried using passwords like 'wzyijmX6ql$V7ghHCXWWtmi' and it's not strong enough apparently.

**Comment by silentworks** [comment]
Is this stopping you from creating the database? or is it just a warning indicator?

**Reply by IzMichael** [reply]
It's stopping me from creating the database

**Comment by inian** [comment, marked as answer]
Hmm I entered that password in our project create page and it seems to pass the validation @IzMichael. Can you check if any requests are failing? Thanks!

![CleanShot 2021-07-30 at 13 36 49](https://user-images.githubusercontent.com/2155545/127605863-a034d770-baf8-403c-baa2-9f8254f9d527.png)

**Reply by IzMichael** [reply]
![image](https://user-images.githubusercontent.com/57115702/127833688-74bf152b-569d-4216-882a-50154c1ca5e0.png)
Yes, a request is failing

**Reply by IzMichael** [reply]
The URL linked just displays this:
![image](https://user-images.githubusercontent.com/57115702/127833869-90b9fc05-d4e0-4524-9a06-d98fe5682841.png)

**Reply by inian** [reply]
When you paste the password in the filed, can you check what the request payload is in the network tab? 
<img width="1116" alt="CleanShot 2021-08-02 at 17 09 49@2x" src="https://user-images.githubusercontent.com/2155545/127835836-345e0222-6a4e-434b-8eb8-885b36f7adca.png">

Opening the url in a new tab is expected to throw an error since GET is not supported at that endpoint, so please ignore that

**Reply by IzMichael** [reply]
![image](https://user-images.githubusercontent.com/57115702/127951856-d947ebe6-22b1-4a1f-93cd-b7cb9af0382c.png)
![image](https://user-images.githubusercontent.com/57115702/127951861-b5831ff6-7f4d-4dbb-840a-1bf887aa7f32.png)

**Reply by IzMichael** [reply]
For the record, I am unable to create or modify an organization as well

**Reply by inian** [reply]
Lets discuss here https://github.com/supabase/supabase/issues/2696

**Comment by IzMichael** [comment]
My view:
![image](https://user-images.githubusercontent.com/57115702/127833623-cddf5aa9-5147-4dee-bb28-277c37fd459f.png)

**Comment by cobblestone-fsh** [comment]
Is this issue related to your issue? It says that BitDefender (and maybe some other programs) falsely have a problem with supabase domains.

https://github.com/supabase/supabase/issues/2696

**Reply by IzMichael** [reply]
Yeah
